irods/irods_runtime_init.yml

```diff
@@ -151,3 +151,18 @@
         stdin: "{{ _irods_admin_password }}"
       register: results
       changed_when: results.stdout == 'scheduled trash removal'
+  
+    - name: Ensure that the starting collections get UUIDs
```

Comment:
It would make more sense to place this task near the other tasks manipulating collections. Maybe it could be placed after the task `Ensure public group's trash is correct`.